app/scripts/configs/available-track-types.js

```diff
@@ -21,20 +21,24 @@ import { DATATYPE_TO_TRACK_TYPE } from '.';
  */
 export const AVAILABLE_TRACK_TYPES = (datatypes, orientation) => {
   const datatypesToTrackTypes = DATATYPE_TO_TRACK_TYPE(orientation);
+  let availableTrackTypes = new Set(
+    Object.values(datatypesToTrackTypes)
+      .flatMap(x => x)
```

Comment:
I've never heard of `flatMap`! Cool stuff

Reply:
Oh yeah, once you know about flatMap it's hard to imagine living without it ;-)